### DISCUSSION TITLE
Follow conventional commits in blueprints?

### DISCUSSION THREAD

**Discussion by PabloVasconez** [Blueprints-org/blueprints#347] [Ideas]
It is perhaps a good idea to follow conventional commits in our repo. Let me know what you think! @RamiEvans @bjlvandergaag @egarciamendez @MaartenBW. 

See link for explanation and examples:
[https://www.conventionalcommits.org/en/v1.0.0/](https://www.conventionalcommits.org/en/v1.0.0/)

**Comment by egarciamendez** [comment]
Sounds like an interesting idea. Worth thinking about it.
